hub-charts/ea-hub/values.yaml

```diff
@@ -30,6 +30,14 @@ jupyterhub:
     memory:
       guarantee: 2G
       limit: 4G
+    lifecycleHooks:
+      postStart:
+        exec:
+          command:
+            - "sh"
+            - "-c"
+            - >
+              gitpuller https://github.com/earthlab-education/earth-analytics-fall-2018 master homework-templates;
```

Comment:
@jlpalomino  this is where we specify the repo that we wish to pull down. `homework-templates` in the example above is the name of the directory where files will live. we can call that whatever you'd like to call it!!

Reply:
shall we wait to merge until we have this changed to the right repo for this year's course?

Reply:
@lwasser @kcranston I just created this repo for homework templates: https://github.com/earthlab-education/earth-analytics-bootcamp-fall-2019

Reply:
@lwasser Your repo from last fall was public, but I made this one private. Let me know if I should change it to public.

Reply:
I think this will have to be public in order for nbgitpuller to be able to read it

Reply:
OK no problem. I have made it public now, as it only contains the template that I gave the students anyway.

Reply:
totally i kept it publich @jlpalomino and just shared lessons we did in class or whatever they needed for class there.